docs/Xeltalliv/simple3D.md

```diff
@@ -269,13 +270,13 @@ For each vertex, the UV texture coordinates at it are calculated as follows:
 
 ### Pixel Colors
 For each pixel, the final color calculation goes as follows:
-1. texture is read or white if texture is not provided
-2. multipled by interpolated vertex color RGBA
+1. texture is read, blank if texture is not provided
+2. multiplied by interpolated vertex color RGBA
```

Comment:
It is important that it's exactly white (RGBA 1,1,1,1) when no texture is provided, because this color is used in all the further computations. The word "blank" is ambiguous and can also mean black (RGBA 0,0,0,1) or transparent (RGBA 0,0,0,0).